presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraMetadata.java

```diff
@@ -351,4 +358,10 @@ public Optional<ConnectorOutputMetadata> finishInsert(ConnectorSession session,
     {
         return Optional.empty();
     }
+
+    @Override
+    public String normalizeIdentifier(ConnectorSession session, String identifier)
+    {
+        return caseSensitiveNameMatchingEnabled ? identifier : identifier.toLowerCase(ENGLISH);
```

Comment:
What if the input is unicode and not english? should use 
identifier.toLowerCase(Locale.ROOT)